### ISSUE TITLE
JSON Parse Error When Setting PRONTO_PHPCS_STANDARD to 'phpcs.xml'

### ISSUE THREAD

**Issue by taufek** [ellisv/pronto-phpcs#4]
I'm getting below error when running pronto with pronto-phpcs while overriding the `standard` flag to 'phpcs.xml' file.

```
> PRONTO_BITBUCKET_USERNAME=myuser@test.com \
  PRONTO_BITBUCKET_PASSWORD=mypassword \
  PRONTO_PHPCS_STANDARD=phpcs.xml \
  pronto run -f bitbucket_pr -c origin/master

/usr/lib/ruby/2.1.0/json/common.rb:155:in `parse': 757: unexpected token at 'Time: 237ms; Memory: 4Mb (JSON::ParserError)
'
        from /usr/lib/ruby/2.1.0/json/common.rb:155:in `parse'
        from /var/lib/gems/2.1.0/gems/pronto-phpcs-0.2.1/lib/pronto/phpcs.rb:39:in `run_phpcs'
        from /var/lib/gems/2.1.0/gems/pronto-phpcs-0.2.1/lib/pronto/phpcs.rb:28:in `inspect'
        from /var/lib/gems/2.1.0/gems/pronto-phpcs-0.2.1/lib/pronto/phpcs.rb:18:in `block in run'
        from /var/lib/gems/2.1.0/gems/pronto-phpcs-0.2.1/lib/pronto/phpcs.rb:18:in `map'
        from /var/lib/gems/2.1.0/gems/pronto-phpcs-0.2.1/lib/pronto/phpcs.rb:18:in `run'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto/runners.rb:20:in `block in run'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto/runners.rb:13:in `each'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto/runners.rb:13:in `run'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto.rb:64:in `run'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto/cli.rb:61:in `block in run'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto/cli.rb:60:in `chdir'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/lib/pronto/cli.rb:60:in `run'
        from /var/lib/gems/2.1.0/gems/thor-0.19.4/lib/thor/command.rb:27:in `run'
        from /var/lib/gems/2.1.0/gems/thor-0.19.4/lib/thor/invocation.rb:126:in `invoke_command'
        from /var/lib/gems/2.1.0/gems/thor-0.19.4/lib/thor.rb:369:in `dispatch'
        from /var/lib/gems/2.1.0/gems/thor-0.19.4/lib/thor/base.rb:444:in `start'
        from /var/lib/gems/2.1.0/gems/pronto-0.9.5/bin/pronto:6:in `<top (required)>'
        from /usr/local/bin/pronto:23:in `load'
        from /usr/local/bin/pronto:23:in `<main>'
```

Really appreciate if someone can help me with this issue.

It works fine when I removed PRONTO_PHPCS_STANDARD environment variable.

The `phpcs.xml` does exists and it works fine when I ran with `phpcs` command. The `phpcs.xml` content can be found in below link:

https://gist.github.com/paulund/8aacc8e8216c21136c182576fc078ec4

**Comment by taufek** [comment]
I found the root cause. When we run below `phpcs` command:

```
phpcs --report=json --standard=phpcs.xml myfile.php
```

The json output has trailing time taken report.

```
{"totals":{"errors":0,"warnings":0,"fixable":0},"files":{}}Time: 45ms; Memory: 4Mb
```

That causes the code to throw an error.

**Comment by taufek** [comment]
I've created a fix in https://github.com/EllisV/pronto-phpcs/pull/5.

**Comment by ellisv** [comment]
Fixed within #5
